### ISSUE TITLE
Text flow functionallity

### ISSUE THREAD

**Issue by Licymnius** [vvvv/SVG#572]
Flowing text is not supported by the library for now. Is it going to be implemented soon?

**Comment by mrbean-bremen** [comment]
By text flow, you mean line wrapping? That is not an SVG 1.1 feature - currently only SVG 1.1 is (partly) implemented. Though if someone could implement this, we would add it happily.
Is that some critical feature you need to use the library?

**Comment by Licymnius** [comment]
Yes, I mean text wrapping. In fact text box. It's not critical, but it could be useful in my project. Maybe I'll implement it in your project, I'll think about it.

**Comment by mrbean-bremen** [comment]
> Maybe I'll implement it in your project. 

Thanks, that would be great!
